### DISCUSSION TITLE
How to enable the timeline view?

### DISCUSSION THREAD

**Discussion by karim-tao** [saade/filament-fullcalendar#183] [Q&A]
Hi,
I'm trying to activate the timeline, I've searched on the internet, and made many attempts. But I can't activate the timeline view.
Can someone help me?

I would like to have a result like this.
![Screenshot 2024-05-10 at 11-36-23 Demos FullCalendar](https://github.com/saade/filament-fullcalendar/assets/30530408/bafe9517-421c-4b5f-8e59-385e460c7fca)

This is my Provider configuration.
<img width="477" alt="Screenshot 2024-05-10 alle 11 39 54" src="https://github.com/saade/filament-fullcalendar/assets/30530408/2c4b567f-76b9-46f3-847a-58dab510e598">

This is my calendar widget.
<img width="843" alt="Screenshot 2024-05-10 alle 11 41 13" src="https://github.com/saade/filament-fullcalendar/assets/30530408/6b2fb95d-06af-4c8b-aecd-5c2965c7b6df">

I tried to use the trial key "CC-Attribution-NonCommercial-NoDerivatives".  But i had errors in browser console.

Do you have any examples you can give me, or alternatives to get a result similar to this?

Thank you

**Comment by jncalderon** [comment]
@uncrackable404 do you refer "month" view? timeline view it's a premium feature that you need pay for a licence

**Comment by aurawindsurfing** [comment]
@uncrackable404 did you figure out how to use premium version of the calendar?
